src/main/java/seedu/address/flashcard/Question.java

```diff
@@ -8,8 +10,11 @@
     public static final String MESSAGE_CONSTRAINTS =
             "Questions should only contain alphanumeric characters and spaces, and it should not be blank";
 
-    String getQuestion();
+    List<String> getQuestion();
 
     String getOnlyQuestion();
```

Comment:
Should we should rename these methods as `getQuestionAsList` and `getQuestionAsString`? Might be less confusing in the future.

Reply:
getQuestion now returns String, I think the one you are looking at is outdated

Reply:
alright!